core/sql/regress/compGeneral/EXPECTED042

```diff
@@ -1977,25 +1977,31 @@ NUM_HITS    NUM_PARAMS
 ----------  ----------
 
          0           1
+         0           1
+         0           1
+         0           1
+         0           2
          0           2
+         0           2
+         0           2
+         0           3
          0           3
+         0           4
          0           5
          1           1
          1           1
          1           2
          1           2
          1           3
+         1           3
```

Comment:
Same number of queries as before. What's happening is that some of the cache entries are not matching up due to differences in CQDs. That's not surprising as the bug in 1494 prevented the toggling of CQDs in some circumstances. In this particular test suite, the metadata queries get cached very early on and the increase in cache usage stops. (I found this out by peppering a copy of the test script with selects against querycache.) So it is up-front growth. Of course if we wish, we could investigate further and perhaps add or delete some of the CQDs in the metadata code to improve the entry sharing.